packages/terra-toggle/src/terra-dev-site/doc/example/DefaultToggle.jsx

```diff
@@ -19,7 +19,10 @@ class ToggleDefault extends React.Component {
     const { isOpen } = this.state;
     return (
       <div>
-        <IconInformation onClick={this.handleOnClick} />
+        <IconInformation onClick={this.handleOnClick} aria-expanded={isOpen} />
```

Comment:
Let's add the aria-controls and id similar to `packages/terra-toggle/src/terra-dev-site/doc/example/AnimatedToggle.jsx`

Reply:
Resolved in 3478f79